Gaijinizer patch/Map229.txt

```diff
@@ -2,33 +2,36 @@
 > BEGIN STRING
 \\{めっちゃ暇ーーーーーー！！！
 > CONTEXT: Map229/events/5/pages/0/8/Dialogue/enemy11
-\\{My spare time!!!#MTLed
+\\{I'm boooooored!!!#StrateLocalization
 > END STRING
 
 > BEGIN STRING
 久々に１階に降りてきたと思ったら、第一声がそれ…？
 > CONTEXT: Map229/events/5/pages/0/11/Dialogue/enemy19
-If you think it has come down to the first floor
-after a long time, the first voice is that ...?#MTLed
+If you've wanted to come down to the first floor
+for so long, why is that the first thing you
+say...?#StrateUnsure
 > END STRING
 
 > BEGIN STRING
 だって………\\N[1]は朝から依頼で出かけてるし
 
 ひとりでゲームやってもつまんないし………
 > CONTEXT: Map229/events/5/pages/0/13/Dialogue/enemy11
-Because...... \\N[1] has been out for a request
-from the morning. I'm not alone in playing games
-.........#MTLed
+Because...... \\N[1] goes to deal with jobs in
+the morning.
+
+And I don't feel like playing games alone......#Strate
 > END STRING
```

Comment:
Think this means boring? 
![image](https://user-images.githubusercontent.com/48231102/107165157-a24e5780-6977-11eb-8f6a-d02b601fe9d7.png)

I'd rephrase it like this:
```
Because... \\N[1] goes to deal with jobs in
the morning.

And it's boring playing game by myself...
```